examples/vmseries_ha/README.md

```diff
@@ -0,0 +1,284 @@
+  "spoke2-vm" = "192.168.2.2"
+}
+vmseries_private_ips = {
+  "fw-vmseries-01" = {
+    "0" = "10.10.11.2"
+    "1" = "10.10.10.2"
+    "2" = "10.10.12.2"
+    "3" = "10.10.13.2"
+  }
+  "fw-vmseries-02" = {
+    "0" = "10.10.11.3"
+    "1" = "10.10.10.3"
+    "2" = "10.10.12.3"
+    "3" = "10.10.13.3"
+  }
+}
+vmseries_public_ips = {
+  "fw-vmseries-01" = {
+    "0" = "<UNTRUST_PUBLIC_IP>"
+    "1" = "<MGMT_PUBLIC_IP>"
+  }
+  "fw-vmseries-02" = {
+    "0" = "<UNTRUST_PUBLIC_IP>"
+    "1" = "<MGMT_PUBLIC_IP>"
+  }
+}
+```
+
+
+## Post build
+
+Connect to the VM-Series instance(s) via SSH using your associated private key and check if the bootstrap process if finished successfuly and then set a password :
+  - Please allow for up to 10-15 minutes for the bootstrap process to finish
+  - The key output you should check for is "Auto-commit Successful"
+
+```
+ssh admin@x.x.x.x -i /PATH/TO/YOUR/KEY/id_rsa
+Welcome admin.
+admin@PA-VM(active)> show system bootstrap status
+
+Bootstrap Phase               Status         Details
+===============               ======         =======
+Media Detection               Success        Media detected successfully
+Media Sanity Check            Success        Media sanity check successful
+Parsing of Initial Config     Successful     
+Auto-commit                   Successful
+
+admin@PA-VM(active)> configure
+Entering configuration mode
+[edit]                                                                                                                                                                                  
+admin@PA-VM(active)# set mgt-config users admin password
+Enter password   : 
+Confirm password : 
+
+[edit]                                                                                                                                                                                  
+admin@PA-VM(active)# commit
+Configuration committed successfully
+```
+
+## Check access via web UI
+
+Use a web browser to access `https://<MGMT_PUBLIC_IP>` and login with admin and your previously configured password.
+
+## Change the public Loopback public IP Address
+
+For the VM-Series that are backend instance group members of the public-facing loadbalancer - go to Network -> Interfaces -> Loopback and change the value of `1.1.1.1` with the value from the `EXTERNAL_LB_PUBLIC_IP` from the terraform outputs.
+
+In order to successfuly access the web server hosted on `spoke-1-vm` - also reconfigure the two NAT policies that contain the destination address of `1.1.1.1` with the IP address from `EXTERNAL_LB_PUBLIC_IP` in Policies -> NAT -> `no-nat-lb-healthchecks` + `inbound-app1`
+
+## Check traffic from spoke VMs
+
+The firewalls are bootstrapped with a generic `allow any` policy just for demo purposes along with an outboud SNAT policy to allow Inernet access from spoke VMs.
+
+SSH to one of the spoke VMs using GCP IAP and gcloud command and test connectivity :
+
+
+```
+gcloud compute ssh spoke1-vm
```

Comment:
Is user is changing region and prefix, then we need to extend a little bit that command:

```suggestion
gcloud compute ssh <NAME-PREFIX-VALUE-FROM-TFVARS>spoke1-vm --zone=<NAME-OF-THE-ZONE-FROM-TFVARS>
```